resources/puppetlabs/lein-ezbake/template/foss/ext/debian/prerm.erb

```diff
@@ -1,9 +1,16 @@
 #!/bin/sh
 
-if [ "$1" = "failed-upgrade" ]; then
-    invoke-rc.d <%= EZBake::Config[:project] -%> stop || :
+if [ -x "/lib/systemd/system/<%= EZBake::Config[:project] -%>.service ] ; then
+  # Using systemd
+  systemctl --no-reload disable <%= EZBake::Config[:project] -%>.service > /dev/null 2>&1 || :
+  systemctl stop <%= EZBake::Config[:project] -%>.service > /dev/null 2>&1 || :
 else
+  # Using sysv
+  if [ "$1" = "failed-upgrade" ]; then
+    invoke-rc.d <%= EZBake::Config[:project] -%> stop || :
+  else
     if [ -x "/etc/init.d/<%= EZBake::Config[:project] -%>" ] || [ -e "/etc/init/<%= EZBake::Config[:project] -%>.conf" ]; then
-        invoke-rc.d <%= EZBake::Config[:project] -%> stop || exit $?
+      invoke-rc.d <%= EZBake::Config[:project] -%> stop || exit $?
```

Comment:
Interesting that in the sysv case we have `|| exit $?` and for systemd it's just `||:`